src/settings-ui/Settings.UI/App.xaml

```diff
@@ -57,6 +57,11 @@
                 BasedOn="{StaticResource DefaultCheckBoxStyle}"
                 TargetType="controls:CheckBoxWithDescriptionControl" />
             <!--  Other app resources here  -->
+
+            <TransitionCollection x:Key="SettingsCardsAnimations">
+                <EntranceThemeTransition FromVerticalOffset="50"/> <!-- Animates cards when loaded-->
```

Comment:
What is this value based on?

Reply:
From an internal spec.. have if you have some public docs indicating 96 would love to learn where!

Reply:
Oh I saw this video https://youtu.be/iBh2jSdVK-8 and at 0:27 I saw the animation in Settings. Seems like more than 50, at least? But if your source is good then I'll 🤐